updater/scripts/git-download.sh

```diff
@@ -14,20 +14,22 @@ function ghe_greater_equal () {
 }
 
 if ghe_greater_equal "2.11.0" ; then
-    LOG_FILE="/var/log/github-audit.log"
+    # check the two most recent log files
+    CAT_LOG_FILE="zcat -f /var/log/github-audit.log{,.1*}"
     # The "github-audit.log" log file introduced in GHE 2.11.0 is only rolled
     # once a week. This was reported as a bug and is likely fixed in an
     # upcoming version. In the meantime we grep for all log entries that have
     # been written yesterday.
     GREP_YESTERDAY="grep -F '$(date --date='yesterday' +'%b %d')'"
 else
-    LOG_FILE="/var/log/github/audit.log"
+    # check yesterday’s log file
+    CAT_LOG_FILE="zcat -f /var/log/github/audit.log.1*"
     GREP_YESTERDAY="tee"
 fi
 
 echo -e "repository\tuser\tcloning?\trequests/day\tdownload/day [B]"
 
-zcat -f $LOG_FILE.1* |
+eval "$CAT_LOG_FILE" |
```

Comment:
Why do we need eval here? `zcat -f $LOG_FILE` wouldn't work?

Reply:
Yes, it wouldn’t, because the Bash wouldn’t expand the wildcards within `CAT_LOG_FILE` without `eval`.

Reply:
I guessed that. But in that case we can move the $GREP_YESTERDAY into this eval, no?

Reply:
Yep, we should be able to put both `$CAT_LOG_FILE` and `$GREP_YESTERDAY` into the same `eval`. I just thought it might be nicer to keep the commands individual and not have pipe operators within `eval`s and outside of them.

What do you think?

Reply:
All this stuff is just a workaround and the workaround requires both evals. Therefore, I think it should be one eval in the end. As soon as the workaround is not necessary anymore we can get rid of the eval completely.

Reply:
All right, I’ll commit the change 🙂.